artifact/artifact/v1alpha/artifact.proto

```diff
@@ -191,6 +203,10 @@ message UpdateKnowledgeBaseRequest {
   string description = 3;
   // The knowledge base tags.
   repeated string tags = 4;
+  // The knowledge base owner.
+  string owner = 5;
```

Comment:
```suggestion
  string owner_name = 5;
```

We should follow the style used in other [backends](https://github.com/instill-ai/protobufs/blob/6e9050ef9d10eb61cc000e4bf19ce6de95d404e0/vdp/pipeline/v1beta/pipeline.proto#L306). We use `owner_name` to indicate the path `users/<user_id>`, `organizations/<org_id>`, and we use `owner` to embed the owner data.

Reply:
I see.  for now, I will change the first.